src/harness/testcases/WINNF_FT_S_HBT_testcase.py

```diff
@@ -794,4 +794,34 @@ def test_WINNF_FT_S_HBT_9(self):
     self.assertEqual(response[2]['grantId'], grant_ids[2])
     self.assertEqual(response[2]['response']['responseCode'], 500)
     self.assertLessEqual(datetime.strptime(
-        response[2]['transmitExpireTime'],'%Y-%m-%dT%H:%M:%SZ'), datetime.utcnow())
\ No newline at end of file
+        response[2]['transmitExpireTime'],'%Y-%m-%dT%H:%M:%SZ'), datetime.utcnow())
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_HBT_11(self):
+    """Out of sync Grant state between the CBSD and the SAS.
+
+    Returns response code 502 (UNSYNC_OP_PARAM).
+    """
+    # Load a device.
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+
+    # Load grant.json
+    grant_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+
+    # Register the device and get a grant
+    cbsd_ids, grant_ids = self.assertRegisteredAndGranted([device_a], [grant_0])
+
+    # Send a Heartbeat request.
+    heartbeat_request = [{
+        'cbsdId': cbsd_ids[0],
+        'grantId': grant_ids[0],
+        'operationState': 'AUTHORIZED'
+    }]
+    request = {'heartbeatRequest': heartbeat_request}
+    response = self._sas.Heartbeat(request)['heartbeatResponse'][0]
+    # Check the heartbeat response.
```

Comment:
A minor comment. The test spec mentions to verify  that just one heartbeat response is included. Perhaps we can add a check to verify that no more than one response is present.

Reply:
Done